frontend/src/utils/v1/instance.ts

```diff
@@ -19,11 +24,17 @@ import { PlanType } from "@/types/proto/v1/subscription_service";
 export function instanceV1Name(instance: Instance | InstanceResource) {
   const store = useSubscriptionV1Store();
   let name = instance.title;
-  // instance cannot be deleted and activated at the same time.
+  // For unknown or empty instance, we will use the name as the title.
+  if (
+    instance.title === unknownInstance().title ||
+    instance.title === emptyInstance().title
```

Comment:
Can we use `!isValidInstanceName(instance.name)`?

Reply:
Actually, the instance name is correct.

https://github.com/bytebase/bytebase/blob/main/frontend/src/store/modules/v1/database.ts#L339